docs/source/user-guide/polars_llms.md

```diff
@@ -0,0 +1,76 @@
+# Generating Polars code with LLMs
+
+Large Language Models (LLMs) can sometimes return Pandas code or invalid Polars code in their
+output. This guide presents approaches that help LLMs generate valid Polars code more consistently.
+
+These approaches have been developed by the Polars community through testing model responses to
+various inputs. If you find additional effective approaches for generating Polars code from LLMs,
+please raise a [pull request](https://github.com/pola-rs/polars/pulls).
+
+## System prompt
+
+Many LLMs allow you to provide a system prompt that is included with every individual prompt you
+send to the model. In the system prompt, you can specify your preferred defaults, such as "Use
+Polars as the default dataframe library". Including such a system prompt typically leads to models
+consistently generating Polars code rather than Pandas code.
+
+You can set this system prompt in the settings menu of both web-based LLMs like ChatGPT and
+IDE-based LLMs like Cursor. Refer to each application's documentation for specific instructions.
+
+## Enable web search
+
+Some LLMs can search the web to access information beyond their pre-training data. Enabling web
+search allows an LLM to reference up-to-date Polars documentation for the current API.
+
+However, web search is not a universal solution. If a model is confident in a result based on its
+pre-training data, it may not incorporate web search results in its output.
+
+## Reference documentation
```

Comment:
I feel that this RAG based approach should be preferred. Can this get some more weight and maybe be placed above web search paragraph?

I'd also want to add that we have added an AI plugin in our reference guide that is indexed on the documentation.

Reply:
Done